python/rmm/_lib/device_buffer.pyx

```diff
@@ -290,6 +290,13 @@ cdef class DeviceBuffer:
     cdef void* c_data(self):
         return self.c_obj.get()[0].data()
 
+    cdef unique_ptr[device_buffer] release(self):
```

Comment:
All of the other cdef functions start with `c_`, may as well follow the trend here.